docs/user_guide/tutorial.md

```diff
@@ -294,7 +295,7 @@ local_fraction_naming_template_str = 'sec<n>_fraction'
 
 In Mu et al. (2022), three GNFR sectors were downscaled including traffic, residential heating and shipping. Denby et al. (2024) extended this list to include aviation and off road emission sources. In this tutorial, we will use the simpler setup by Mu et al.
```

Comment:
It would be nice to link the citations either directly to the publication or to the "references" header. I'm not sure which option would be better.

Also, either a short explanation of what "GNFR" is, or link to an explanation I think would be useful.